src/gimelstudio/core/node/node.py

```diff
@@ -45,6 +45,16 @@ def _WidgetEventHook(self, idname, value, render):
         """
         self.NodeWidgetEventHook(idname, value)
         self.SetEditedFlag(True)
+
+        # All nodes except input and output can be muted.
+        # Nodes with multiple inputs should pass input of first one.
+        # Is checked when property changes, shouldn't be here.
+        if self.IsMuted():
+            self.SetPropMute(True)
+            return  # Return input image, how ? self.nodegraph.parent.parent.Render() is I guess output of current node.
```

Comment:
It isn't ideal (it is quite messy), however this is how to get the rendered image
```python
self.nodegraph.parent.parent.renderer.GetRender()
```